docs/guides/runtime-options-overview.mdx

```diff
@@ -61,50 +60,50 @@ Scroll to see the full table.
 |                      | skip_reset_qubits        |                         | `True`/`False`                                                                                   | `False`                 |                    |
 | [environment](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EnvironmentOptions)          |                          |     |             |                                  | environment          |
-|                      | log_level                |                         |`DEBUG`/`INFO`/`WARNING`/`ERROR`/`CRITICAL`                 |`WARNING`                          |                      |
 |                      | callback                 |                         |Callable function that receives Job ID and Job result.                             |`None`                           |                      |
 |                      | job_tags                 |                         |List of tags                                                                     |`None`                           |                      |
+|                      | log_level                |                         |`DEBUG`/`INFO`/`WARNING`/`ERROR`/`CRITICAL`                 |`WARNING`                          |                      |
 |                      | private                  |                         |`True`/`False`                                                                      |`False`                           |                      |
 | [execution](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ExecutionOptionsV2)            |                          |                         |               |                           | execution            |
 |                      | init_qubits              |                         | `True`/`False`                                                                         | `True`                    |                      |
 |                      | rep_delay                |                         | Value in the range supplied by `backend.rep_delay_range`.       | Given by `backend.default_rep_delay`.|                      |
 | [max_execution_time](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions)   |                          |       | Integer number of seconds in the range  [1, 10800]    | `10800` (3 hours)   | max_execution_time   |
 | [resilience](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2)           |                          |      |                               |                      | resilience    |
-|                      | [layer_noise_learning](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.LayerNoiseLearningOptions)     |      | `True`/`False`                | `True`               |      |
+|                      | layer_noise_learning     |      | `True`/`False`                | `True`               |      |
```

Comment:
`layer_noise_learning` is a sub-category and doesn't have `True/False` (or any) values, similar to `zne` and `pec`.

In addition, for the sub-categories, shouldn't we link to the API reference like what the main categories (e.g. `resilience`) have?

Reply:
Yeah I think it'd be useful and also consistent with the main categories.